### PR TITLE
map inside locationPage

### PR DESCRIPTION
**Pull request by gilbaadani**
scr

### CODE REVIEW

File: src/components/Maps/MapWithMarkers.js [hunk 1]
```diff
@@ -42,15 +42,26 @@ const MapWithAMarkerClusterer = compose(
   </GoogleMap>
 ));
 
-const MapWithMarkers = ({ onMarkerClick, dataMarkers, lat, lng }) => (
-  <MapWithAMarkerClusterer onMarkerClick={onMarkerClick} markers={dataMarkers} lat={lat} lng={lng} />
+const MapWithMarkers = ({ onMarkerClick, dataMarkers, lat, lng, defaultZoom }) => (
+  <MapWithAMarkerClusterer
```
Comment (Alaev) [inline src/components/Maps/MapWithMarkers.js]: Why `MapWithAMarkerClusterer` here? why cant we use single marker?

File: src/pages/Location/Location.js [hunk 2]
```diff
@@ -21,16 +23,24 @@ const ReviewsWrapper = styled.div`
...
-      <Menu grid="2fr" />
-      <ReviewsWrapper>
-        <ReviewList data={reviews} />
-      </ReviewsWrapper>
-    </LocationWrapper>
-  </div>
-);
+
+const location = () => {
+  const currentLocationId = window.location.href.substr(window.location.href.lastIndexOf('/') + 1);
```
Comment (Alaev) [inline src/pages/Location/Location.js]: why do we need this function here?

File: src/pages/Location/Location.js [hunk 3]
```diff
@@ -21,16 +23,24 @@ const ReviewsWrapper = styled.div`
...
+  const locateLocationById = element => currentLocationId === element.id;
+  const myLocation = [data.cards.find(locateLocationById)];
```
Comment (Alaev) [inline src/pages/Location/Location.js]: same here

File: src/components/Maps/MapWithMarkers.js (hunk 1 above)
Comment (gilbaadani) [reply src/components/Maps/MapWithMarkers.js]: MapWithAMarkerClusterer receives an array of objects and makes one marker per object. we send him an array with one object.

File: src/pages/Location/Location.js (hunk 2 above)
Comment (gilbaadani) [reply src/pages/Location/Location.js]: putting currentLocationId outside of location made an asynchronously problem. when i presesd a card it has been shown an error in the console screen. putting this inside location solved it

File: src/pages/Location/Location.js (hunk 3 above)
Comment (gilbaadani) [reply src/pages/Location/Location.js]: same here. currentLocationId  is to find the ID , myLocation  is to find the location of the card which was pressed

**Pull request merged**